python/ray/serve/deployment.py

```diff
@@ -7,14 +7,14 @@
 from ray.dag.class_node import ClassNode
 from ray.dag.dag_node import DAGNodeBase
 from ray.dag.function_node import FunctionNode
-from ray.serve._private.config import DeploymentConfig, ReplicaConfig
+from ray.serve._private.config import InternalDeploymentConfig, ReplicaInitInfo
 from ray.serve._private.constants import MIGRATION_MESSAGE, SERVE_LOGGER_NAME
 from ray.serve._private.usage import ServeUsageTag
 from ray.serve._private.utils import DEFAULT, Default, guarded_deprecation_warning
-from ray.serve.config import AutoscalingConfig
+from ray.serve.config import AutoscalingConfig, RayActorOptionsConfig
 from ray.serve.context import _get_global_client
 from ray.serve.handle import RayServeHandle, RayServeSyncHandle
-from ray.serve.schema import DeploymentSchema, RayActorOptionsSchema
+from ray.serve.schema import ApplyServeDeploymentModel
```

Comment:
```suggestion
from ray.serve.schema import ApplyDeploymentModel
```

"Serve" is redundant (everything is Serve) and the base is called `BaseDeploymentModel`